test/MOI_wrapper.jl

```diff
@@ -49,5 +53,5 @@ const config = MOIT.TestConfig(atol=1e-4, rtol=1e-4, optimal_status=MOI.LOCALLY_
                 "solve_result_index", # no support for `MOI.ConstraintPrimal` atm
                 "update_dimension_nonnegative_variables", # currently no support for ConstraintFunction 
                ]
-    MOIT.unittest(bridged, config, exclude)
+    MOIT.runtests(optimizer, config; exclude=exclude)
```

Comment:
Run the tests in a caching optimizer, that should fix a lot of the `loadfromstring!` errors.

```Julia
MOI.Utilities.CachingOptimizer(
    MOI.Utilities.UniversalFallback(MOI.Utilities.Model{Float64}()),
    optimizer,
)
```